zim/gui/__init__.py

```diff
@@ -1836,7 +1836,7 @@ def statusbar_element(string, size):
 			return label
 
 		# specify statusbar elements right-to-left
-		self.statusbar_style_label = statusbar_element('<style>', 100)
+		self.statusbar_style_label = statusbar_element('<styles>', 110)
```

Comment:
status bar space was too small... it might be dynamic or so but I just made it a little wider